modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -3152,6 +3161,54 @@ export class Wallet implements IWallet {
     }
   }
 
+  /**
+   * Signs a transaction from a TSS ECDSA wallet using external signer.
+   *
+   * @param params signing options
+   */
+  private async signTransactionTssExternalSignerECDSAMPCv2(
+    coin: IBaseCoin,
+    params: WalletSignTransactionOptions = {}
+  ): Promise<TxRequest> {
+    let txRequestId = '';
+    if (params.txRequestId) {
+      txRequestId = params.txRequestId;
+    } else if (params.txPrebuild && params.txPrebuild.txRequestId) {
+      txRequestId = params.txPrebuild.txRequestId;
+    } else {
+      throw new Error('TxRequestId required to sign TSS transactions with External Signer.');
+    }
+
+    if (!params.customMPCv2Round1GenerationFunction) {
+      throw new Error('Generator function for MPCv2 Round 1 share required to sign transactions with External Signer.');
+    }
+
+    if (!params.customMPCv2Round2GenerationFunction) {
+      throw new Error('Generator function for MPCv2 Round 2 share required to sign transactions with External Signer.');
+    }
+
+    if (!params.customMPCv2Round3GenerationFunction) {
+      throw new Error('Generator function for MPCv2 Round 3 share required to sign transactions with External Signer.');
+    }
+
+    try {
+      assert(this.tssUtils, 'tssUtils must be defined');
+      const signedTxRequest = await this.tssUtils.signEcdsaMPCv2TssUsingExternalSigner(
+        {
+          txRequest: txRequestId,
+          prv: '',
```

Comment:
Nit: we should allow undefined `prv` if the custom signing functions are passed. Seems weird to pass an empty string.

Reply:
Ya, its mostly because its using the wrong type. i will fix it